imsc1_1/ttml/initial/initial001.ttml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Not sure the initial tests are supposed to be in this branch? They seem to be in a different pull request too.